biz.aQute.bndlib/src/aQute/bnd/service/result/Result.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2016 René Perschon <rperschon85@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aQute.bnd.service.result;
+
+import java.util.Optional;
+import java.util.function.Function;
```

Comment:
This seems to be here due to incorrect references in the javadoc. The javadoc should reference FunctionWithException.

Reply:
k